canvas_modules/common-canvas/src/palette/palette-content-list-item.jsx

```diff
@@ -224,6 +225,12 @@ class PaletteContentListItem extends React.Component {
 		return highlightedElements;
 	}
 
+	keyPressEvent(e) {
+		if (e.key === " " || e.code === "Space" || e.keyCode === 32) {
```

Comment:
I don't think it is necessary to check for e.key here. The other two are enough.

Reply:
e.key condition has been removed.